extension/package.json

```diff
@@ -7,7 +7,7 @@
 	"version": "2.4.0",
 	"license": "GPL-3.0",
 	"engines": {
-		"vscode": "^1.46.0"
+		"vscode": "^1.55.0"
```

Comment:
We need to upgrade the version of vscode since InlineValuesProvider is not introduced in "1.46.0".